rosette/rbl/rosette/space.rbl

```diff
@@ -112,14 +112,14 @@
     [[channels unification_ptrns product_ptrns persistent] location]
         ;;; the channel and the pattern of incoming messages to look for are destructured and bound
     [_ (seq
-        (print (->symbol "DEBUG: Calling consume("))
-        (print channels)
-        (print (->symbol ", "))
-        (print unification_ptrns)
-        (print (->symbol ", "))
-        (print product_ptrns)
-        (print (->symbol ")"))
-        (p "")
+        (if flag-verbose (print (->symbol "DEBUG: Calling consume(")))
+        (if flag-verbose (print channels))
+        (if flag-verbose (print (->symbol ", ")))
+        (if flag-verbose (print unification_ptrns))
+        (if flag-verbose (print (->symbol ", ")))
+        (if flag-verbose (print product_ptrns))
+        (if flag-verbose (print (->symbol ")")))
+        (if flag-verbose (p ""))
```

Comment:
Maybe just check the flag once in each block of `print` expressions?

Reply:
I thought about that, but couldn't seem to get the syntax correct.

Reply:
```scheme
(if flag-verbose
  (seq
    (print ...)
    (print ...)))
```